### DISCUSSION TITLE
Ability to add notes to a container

### DISCUSSION THREAD

**Discussion by deanfourie1** [louislam/dockge#580] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The ability to add notes to individual container within the UI.

### ✔️ Solution

Just a text box that can be used to add notes for container specific commands or notes to remember.

### ❓ Alternatives

Nothin

**Comment by trythatagain** [comment]
Why don't you just add comments to your compose.yml?

**Reply by TomaszCwajda-hub** [reply]
these are two different things, a comment in the source and a comment in the instance, aren't they?

**Reply by deanfourie1** [reply]
Yea  nah this is not what i mean, 

What I mean is, adding visual comments to the UI, to track disks, or to identify disks etc.

**Comment by Nanorithm** [comment]
+1 for this, I'd love to have a note section and not have to clutter my compose files with junk comments. 

Lots of times I have containers that aren't working and I want to stop them and come back to them later to fix but I end up forgetting what I was doing or why I had a container stopped in the first place.. Obviously I know there are other solutions like keeping my own separate notes, but a note section would be a nice (and lightweight) QoL imo.

Something like an optional use textbox that creates a .note file in the stack folder would be plenty enough for me.

**Comment by HelloMihai** [comment]
+1

this would be really useful.  Just as we have an edit for the compose yaml we should have readme md that can be editable in the stack.  Very useful.

**Comment by luckman212** [comment]
+1 from me too. A purposeful field for notes/URLs etc would be much preferred over contaminating the compose YAML files.

**Comment by Aerya** [comment]
Hi, added to my Enhanced fork.

**Comment by luckman212** [comment]
@Aerya first time hearing about this fork. Thanks, I'll check it out.
